NonSucking.Framework.Serialization/Serializers/CustomMethodCallSerializer.cs

```diff
@@ -19,6 +19,33 @@ namespace NonSucking.Framework.Serialization
     [StaticSerializer(0)]
     internal static class CustomMethodCallSerializer
     {
+        private static IMethodSymbol? GetMethod(NoosonGeneratorContext context, MemberInfo property, ITypeSymbol? customType, string methodName, bool isClassAttribute)
+        {
+            ITypeSymbol typeToCallOn;
+            bool matchAdditionalParam = false;
+            if (customType is not null)
+            {
+                typeToCallOn = customType;
+
+                matchAdditionalParam = context.WriterTypeName is not null;
+            }
+            else if (isClassAttribute || (property.Symbol is not IPropertySymbol && property.Symbol is not IFieldSymbol))
```

Comment:
```suggestion
            else if (isClassAttribute 
                || (property.Symbol is not IPropertySymbol 
                    && property.Symbol is not IFieldSymbol))
```
This line seems to be a bit long to be kept as a single one